tests/test.py

```diff
@@ -8762,15 +8762,6 @@ def test_invalid_specified_values(self):
         self.assertTrue(found, "Not found any expected exception")
 
 
-class TestPanicRecover(AcraCatchLogsMixin, BaseTestCase):
```

Comment:
no more test? 🥺

Reply:
it was one known case that caused panics. I don't know other cases to put here *( We can leave this test empty, but IMHO, we can write it one more time when we will need it.

Reply:
i thought this PR means that panics won't happen for null:text cast, so the test should always pass now.

Reply:
And after this fix it doesn't panic anymore )

Reply:
> i thought this PR means that panics won't happen for null:text cast, so the test should always pass now.

oh, you wrote it before me)) 10 sec difference) Yes, it fixes and doesn't panic anymore

Reply:
okay, thank you for the explanation! great job!